roles/openshift_examples/files/examples/v1.3/cockpit-templates/enterprise/openshift-cockpit.template

```diff
@@ -0,0 +1,170 @@
+         "value": "false"
+      }
+   ],
+   "objects": [
+      {
+         "kind":"DeploymentConfig",
+         "apiVersion":"v1",
+         "metadata":{
+            "name":"openshift-cockpit",
+            "labels":{
+               "name":"openshift-cockpit"
+            }
+         },
+         "spec":{
+            "replicas":1,
+            "selector":{
+               "name":"openshift-cockpit"
+            },
+            "template":{
+               "metadata":{
+                  "labels":{
+                     "name":"openshift-cockpit"
+                  }
+               },
+               "spec":{
+                  "containers":[{
+                    "name": "openshift-cockpit",
+                    "image": "cockpit/kubernetes",
+                    "ports":[{
+                        "containerPort":9090,
+                        "protocol":"TCP"
+                     }],
+                    "env":[
+                      {
+                        "name": "OPENSHIFT_OAUTH_PROVIDER_URL",
+                        "value": "${OPENSHIFT_OAUTH_PROVIDER_URL}"
+                      },
+                      {
+                        "name": "OPENSHIFT_OAUTH_CLIENT_ID",
+                        "value": "${OPENSHIFT_OAUTH_CLIENT_ID}"
+                      },
+                      {
+                        "name": "KUBERNETES_INSECURE",
+                        "value": "${KUBERNETES_INSECURE}"
+                      },
+                      {
+                        "name": "KUBERNETES_CA_DATA",
+                        "value": "${KUBERNETES_CA_DATA}"
+                      },
+                      {
+                        "name": "COCKPIT_KUBE_INSECURE",
+                        "value": "${COCKPIT_KUBE_INSECURE}"
+                      },
+                      {
+                        "name": "REGISTRY_HOST",
+                        "value": "${REGISTRY_HOST}"
+                      },
+                      {
+                        "name": "REGISTRY_ONLY",
+                        "value": "${REGISTRY_ONLY}"
+                      }]
+                  }]
+               }
+            }
+         }
+      },
+      {
+         "kind":"Service",
+         "apiVersion":"v1",
+         "metadata":{
+            "name":"openshift-cockpit",
+            "labels":{
+               "name":"openshift-cockpit"
+            }
+         },
+         "spec":{
+             "type": "ClusterIP",
+             "ports": [{
+                 "name": "legacy",
+                 "protocol": "TCP",
+                 "port": 9000,
+                 "targetPort": 9090
+             },
+             {
+                 "name": "http",
+                 "protocol": "TCP",
+                 "port": 80,
+                 "targetPort": 9090
+             },
+             {
+                 "name": "https",
+                 "protocol": "TCP",
+                 "port": 443,
+                 "targetPort": 9090
+             }],
```

Comment:
Does port 9000 conflict with the locally running cockpit instance? 

Should both http and https both be pointing to targetPort 9090?